src/diffusers/pipelines/controlnet/pipeline_controlnet_sd_xl.py

```diff
@@ -1144,8 +1145,17 @@ def __call__(
 
         # 8. Denoising loop
         num_warmup_steps = len(timesteps) - num_inference_steps * self.scheduler.order
+        is_unet_compiled = is_compiled_module(self.unet)
+        is_controlnet_compiled = is_compiled_module(self.controlnet)
+        is_torch_higher_equal_than_2_1 = version.parse(version.parse(torch.__version__).base_version) >= version.parse(
```

Comment:
```suggestion
        is_torch_higher_equal_2_1 = version.parse(version.parse(torch.__version__).base_version) >= version.parse(
```
Use util torch function here as well